### PR TITLE
Create 2023-10-02-dashboards

### PR DESCRIPTION
**Pull request by kevin-sumo**
Note the deprecation of Classic Dashboards.

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
@kevin-sumo , is this ok to merge now?

**Comment by kevin-sumo** [comment]
Yes, your free to merge if the text look good.

**Comment by jpipkin1** [comment]
@kimsauce, see the message above:
>@[github-merge-queue](https://github.com/apps/github-merge-queue) bot removed this pull request from the [merge queue](https://github.com/SumoLogic/sumologic-documentation/queue/main) due to no response for status checks [40 minutes ago](https://github.com/SumoLogic/sumologic-documentation/pull/3144#event-10529578102)

File: blog-service/2023-10-02-dashboards.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+hide_table_of_contents: true
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - dashboard
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Beginning today, Sumo Logic has removed the ability to create or copy [Classic Dashboards](/docs/dashboards-classic) from within Sumo Logic. We understand that you may have existing classic dashboards that you may still need to access and use. To make this transition as seamless as possible, we have implemented the following:
```
Comment (kimsauce) [inline blog-service/2023-10-02-dashboards.md:12]:
```suggestion
Effective today, Sumo Logic has removed the ability to create or copy [Classic Dashboards](/docs/dashboards-classic) from within Sumo Logic. We understand that you may have existing classic dashboards that you may still need to access and use. To make this transition as seamless as possible, we have implemented the following:
```

**Pull request merged**